### PR TITLE
Add time-varying w

### PR DESCRIPTION
**Pull request by edejong-caltech**

### CODE REVIEW

File: advect.jl [hunk 1]
```diff
@@ -0,0 +1,208 @@
+import ClimaCore:
```
Comment (trontrytel) [inline advect.jl:1]: Do we need this file, or is it a testing leftover?

File: src/tendency.jl [hunk 2]
```diff
@@ -10,15 +10,14 @@ function precompute_aux!(dY, Y, aux, t)
     @. aux.q_liq = TD.liquid_specific_humidity(aux.params, ts)
     @. aux.q_ice = TD.ice_specific_humidity(aux.params, ts)
     @. aux.T     = TD.air_temperature(aux.params, ts)
-
+    aux.w        = Geometry.WVector.(ones(FT, face_space))
+    @. aux.w     = aux.w * aux.w_params.w1 * sin(pi * t / aux.w_params.t1)
 end
```
Comment (trontrytel) [inline src/tendency.jl:15]: Would it make more sense to initialise `w` to zero in `KiD_driver` and here just write

`@. aux.w     = aux.w_params.w1 * sin(pi * t / aux.w_params.t1)`

**Review by trontrytel — COMMENTED** [review]
Left some comments. 

Aside from that, in other CliMA repos we are trying to squash and rebase before merging. This helps keep the history clean. Although, in this small repo, with only the two of us making changes, it's not going to be an issue. Up to you if you want to flex your git skills

File: advect.jl (hunk 1 above)
Comment (edejong-caltech) [reply advect.jl:1]: Leftover, thanks

File: src/tendency.jl (hunk 2 above)
Comment (edejong-caltech) [reply src/tendency.jl:15]: I tried that at first, but it complained about some typing with the Field Vector. Let me try something else.

**Review by trontrytel — APPROVED** [review]

**Pull request merged**